docs/source/configuration/authorization.mdx

```diff
@@ -412,9 +420,201 @@ The response retains the initial request's shape but returns `null` for unauthor
 
 If _every_ requested field requires authentication and a request is unauthenticated, the router generates an error indicating that the query is unauthorized.
 
+<MinVersion version="1.34.0">
```

Comment:
@Geal - what should this minimum router version of GA support for `@policy` be set to?

Reply:
1.35, we're releasing it today